cran-comments.md

```diff
@@ -0,0 +1,14 @@
+## Test environments
+* local R installation, R 4.0.1
+* ubuntu 16.04 (on travis-ci), R 4.0.1
+* win-builder (devel)
+
+## R CMD check results
+
+0 errors | 0 warnings | 0 note
+
+## Response from last CRAN submission comments
+
+- There is no existing references describing the methods in this package. 
+- Added return \value item to Rd
+- Removed examples of unexported functions. 
```

Comment:
Copy and pasting some content from the reviewer in case the reviewer for 
next submission cycle doesn't have this context.

```suggestion
> If there are references describing the methods in your package, please
> add these in the description field of your DESCRIPTION file [...]

- There is no existing references describing the methods in this package. 

> Please add \value to .Rd files regarding exported methods and explain
> the functions results in the documentation. Please write about the
> structure of the output (class) and also what the output means. [...]

- Added return \value item to Rd

> You have examples for unexported functions.
> Please either omit these examples or export these functions. [...]

- Removed examples of unexported functions. 
```